src/main/java/org/akhq/configs/Oidc.java

```diff
@@ -0,0 +1,26 @@
+package org.akhq.configs;
+
+import io.micronaut.context.annotation.ConfigurationProperties;
+import io.micronaut.security.oauth2.endpoint.token.response.OpenIdClaims;
+import lombok.Data;
+
+import java.util.Map;
+
+@ConfigurationProperties("akhq.security.oidc")
```

Comment:
should this be a configuration each?

Reply:
yes I think it must be `@EachProperty`

Reply:
Nope, it is not an array so it has to be `@ConfigurationProperties`

Reply:
my fault ;) 
To be honest, I don't have time to look deeply ;)